src/PageContent/Calendar/DelivererGroupContent.js

```diff
@@ -0,0 +1,133 @@
+import './Content.css';
+import volunteer from '../../icons/volunteer.svg';
+
+class DelivererGroupContent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            edit: false,
+            delivererGroup: this.props.delivererGroup,
+            deliverer1: this.props.delivery.delivererGroup.deliverers[0].name,
+            deliverer2: this.props.delivery.delivererGroup.deliverers[1].name,
+            phone1: this.props.delivery.delivererGroup.deliverers[0].phone,
+            phone2: this.props.delivery.delivererGroup.deliverers[1].phone,
+            email1: this.props.delivery.delivererGroup.deliverers[0].email,
+            email2: this.props.delivery.delivererGroup.deliverers[1].email
+        };
+        this.edit = this.edit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+    edit() {
+        this.setState({
+            edit: true
+        });
+    }
+
+    handleChange(e) {
+        e.preventDefault();
+
+        this.setState({
+            edit: false,
+            deliverer1: e.target.deliverer1.value,
+            phone1: e.target.phone1.value,
+            deliverer2: e.target.deliverer2.value,
+            phone2: e.target.phone2.value
+        });
+
+        // TODO: Save values to firebase
+    }
+
+    render() {
+        return (
+            <div className="wrapper">
+                <img className="content-icon" src={volunteer} alt="volunteer" />
+                <div className="content-wrapper">
+                    {this.props.accountType === AccountType.DELIVERER_GROUP ? (
+                        <h1 className="section-header">Student Deliverers</h1>
+                    ) : (
+                        <h1 className="section-header">Picking Up Donation</h1>
+                    )}
+                    <h2 className="organization">
+                        {this.props.delivererGroup}
+                    </h2>
+                    {!this.state.edit ? (
+                        <div className="content-details-wrapper">
+                            <p className="content-details">
+                                {this.state.deliverer1} ({this.state.phone1})
+                            </p>
+                            <p className="content-details">
+                                {this.state.deliverer2} ({this.state.phone2})
+                            </p>
+                        </div>
+                    ) : (
+                        <div className="content-details-wrapper">
+                            <form
+                                className="edit-dg"
+                                onSubmit={this.handleChange}
+                            >
+                                <div className="input-wrapper">
+                                    <input
+                                        className="content-details deliverer-group-details"
+                                        defaultValue={this.state.deliverer1}
```

Comment:
will probably need to add additional logic to show placeholder text like `"name"`, `"phone (xxx-xxx-xxxx)"`, `"email"` if no deliverers assigned yet